recipe/get_license_family.py

```diff
@@ -0,0 +1,38 @@
+import license_expression
+
+mapping = {
+    "GPL-2.0-or-later": "GPL",
+    "BSD-3-Clause": "BSD",
+    "MIT": "MIT",
+}
```

Comment:
Do we need many more entries here, or did you confirm these are all of the ones that we can fix and/or need fixing?

Reply:
We do. I don't have time to go through all of them, but hope others can help me.

Reply:
Ahhh got it. Many of these can be generated via regex. Have you been able to get a unique list? This repo may help: https://github.com/regro/libcfgraph

Reply:
It would be great if we could centralize this knowledge, I'm thinking of building a mini migrator to fix this in the recipes themselves and this would be super helpful.

Reply:
+1 Whatever mapping is generated can be stored as yaml or json and used in both spots. It doesn't have to be dynamic in that eventually we will catch most of everything.

Reply:
I'll only add `SPDX->license_family` mapping here. For the bot you need `license->SPDX` mapping.